app/ports/management/commands/update-rsync.py

```diff
@@ -0,0 +1,46 @@
+import subprocess
+import json
+
+from django.core.management.base import BaseCommand, CommandError
+
+from parsing_scripts import git_update
+from ports.models import Port, Commit
+
+RSYNC = "rsync://rsync.macports.org/macports//trunk/dports/PortIndex_darwin_16_i386/PortIndex.json"
```

Comment:
First of all, you should be fetching from 18 rather than 16.

I would like to see one single configuration file where this is one of the variables being used.

Over time the origin of this file might change, if nothing else to use a newer OS. If such information is spread in gazzilion of unrelated files, it is super difficult to find it.